lightly/api/api_workflow_download_dataset.py

```diff
@@ -207,4 +207,51 @@ def export_label_box_data_rows_by_tag_name(
 
         """
         tag = self.get_tag_by_name(tag_name)
-        return self.export_label_box_data_rows_by_tag_id(tag.id)
\ No newline at end of file
+        return self.export_label_box_data_rows_by_tag_id(tag.id)
+
+
+    def export_filenames_by_tag_id(
+        self,
+        tag_id: str,
+    ) -> str:
```

Comment:
Does it return a `str` or a `List[str]`?

Reply:
its a `text/plain` and not json so its not a list. Is there a way to check this somehow?